plugin-gradle/README.md

```diff
@@ -457,6 +458,23 @@ black().pathToExe('C:/myuser/.pyenv/versions/3.8.0/scripts/black.exe')
 
 <a name="applying-freshmark-to-markdown-files"></a>
 
+## Protobuf
+
+### buf
+
+`com.diffplug.gradle.spotless.ProtobufExtension` [javadoc](https://javadoc.io/doc/com.diffplug.spotless/spotless-plugin-gradle/6.6.0/com/diffplug/gradle/spotless/ProtobufExtension.html), [code](https://github.com/diffplug/spotless/blob/main/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/ProtobufExtension.java)
+
+```gradle
+spotless {
+  protobuf {
+    // by default the target is every '.proto' file in the project
+    buf()
```

Comment:
This trick should be documented somewhere.

```
configure<SpotlessExtension> {
    protobuf {
        buf().pathToExe(configurations.getByName(BUF_BINARY_CONFIGURATION_NAME).singleFile.absolutePath)
    }
}
```

It can live here and you can link to it from the `buf-gradle-plugin`, or it can live in the docs to `buf-gradle-plugin` and we can link to it from here. But it's too useful to rot alone in the PR comments.

Reply:
Added. I'll be adding a note to the docs on the buf-gradle-plugin as well.